detect_secrets/core/baseline.py

```diff
@@ -34,13 +34,15 @@ def initialize(
         exclude_lines=exclude_lines_regex,
     )
 
-    if os.path.isfile(path):
+    if len(path) > 1 or os.path.isfile(path[0]):
         # This option allows for much easier adhoc usage.
-        files_to_scan = [path]
+        files_to_scan = list()
+        for elem in path:
+            files_to_scan.append(elem)
```

Comment:
Since `path` is already a list, isn't this just:

```
files_to_scan = path
```

?

Reply:
Completely missed this! Sorry, was part of some behavior testing but It's removed now.